### PR TITLE
Fix additional coordinate system information

### PR DESCRIPTION
**Pull request by gmantele**
Following comments (5), (6) and (7) of @msdemlei in the ADQL-2.1 [RFC page](https://wiki.ivoa.net/twiki/bin/view/IVOA/ADQL21RFC#Community%20Review%20by%20Markus%20Demle), here are few fixes about the usage of the coordinate system argument.

Basically:
- remove the section `Utility functions`  ; it just duplicated what was said in section `COORDSYS`, `DISTANCE`, `POINT`, ...
- rename the section `COORDSYS` into `Coordinate system` in order to avoid confusion with the section speaking about the `COORDSYS` function
- rephrase the section `Coordinate system` and include explanations about `DISTANCE`, `CONTAINS` and `INTERSECTS`
- remove explanations from the sections about `DISTANCE`, `CONTAINS` and `INTERSECTS` about automatic coordinates conversion ; reference the section `Coordinate system` instead

### CODE REVIEW

File: ADQL.tex [hunk 1]
```diff
@@ -1490,34 +1490,38 @@ \subsubsection{Datatype functions}
...
-such conversion. If this behavior is desired, the ADQL implementers have to
-allow it through User Defined Functions. Then, it is recommended to define them
-as declared in the \CatalogueUDF{}.
+the appropriate conversion to make these operations possible.
+
+Since version 2.1, this argument is deprecated and has been made optional.
+Future versions of this specification will remove this parameter from the listed
+functions.
+
+Coordinate conversions SHOULD now be explictly asked. The ADQL implementers have
```
Comment (msdemlei) [inline ADQL.tex]: s/asked/requested by the query author

File: ADQL.tex [hunk 2]
```diff
@@ -1490,34 +1490,38 @@ \subsubsection{Datatype functions}
...
+to allow it through User Defined Functions. Then, it is recommended to define
```
Comment (msdemlei) [inline ADQL.tex]: Perhaps we should mention the UDF catalogue more like "an interoperable facility for frame transformations is in preparation as of this writing and is expected to be part of the UDF catalogue".

File: ADQL.tex [hunk 3]
```diff
@@ -1866,7 +1796,7 @@ \subsubsection{CONTAINS}
 is in, or on, the boundary of the second argument and the numeric value 0
 if it is not.
 
-When used as a predicate in the WHERE clause of a query, the numeric return
+When used as a predicate in the WHERE clause of a query, the numeric returned
```
Comment (msdemlei) [inline ADQL.tex]: I think this needs to be "the numeric value (or better: integer) returned"

**Review by msdemlei — APPROVED** [review]
Thanks!  I've made a few minor comments -- feel free to disregard them, and if you honour them, feel free to force-merge afterward, because I'll be on holiday next week and hence I may be really slow to review.

File: ADQL.tex (hunk 1 above)
Comment (gmantele) [reply ADQL.tex]: Done

File: ADQL.tex (hunk 2 above)
Comment (gmantele) [reply ADQL.tex]: Done

File: ADQL.tex (hunk 3 above)
Comment (gmantele) [reply ADQL.tex]: Done

**Pull request merged**